cypress/support/commands.js

```diff
@@ -11,3 +11,19 @@ Cypress.Commands.add('loginAs', (role) => {
     window.localStorage.setItem('ACCESS_TOKEN', `userId=1&${USER_ROLES_MAP[role]}`)
 })
 
+Cypress.Commands.add('login', (user, role) => {
+
+    cy.visit('/')
+        cy.get('.Select.not-valid').should('not.visible')
```

Comment:
This is not part of the login function. Remove these 3 lines.

Reply:
Nice catch, thanks Gerda.